common/stages/Functions/answerfile

```diff
@@ -37,6 +37,14 @@ read_xml_tag()
     sed -ne 's/^.*<'"${TAG}"'\(\s\+[^>]*\)*\s*>\([^<]*\)<\/'"${TAG}"'>.*$/\2/p' <${FILE}
 }
 
+# This assumes the tag is on a single line in the file.
+sanitize_xml_tag()
+{
+    FILE="$1"
+    TAG="$2"
+    sed -i -e 's/\(^.*<'"${TAG}"'\(\s\+[^>]*\)*\s*>\)\([^<]*\)\(<\/'"${TAG}"'>.*$\)/\1SANITIZED\4/' ${FILE}
```

Comment:
For the separator thing, I meant try using "s|||" instead of "s///" since you are matching a slash in the tag close.

`\(\s\+[^>]*\)*` matches spaces, the following `\s*` is probably not needed.

\3 doesn't need to exist since it's not used (the parenthesis can be removed and \4 replaced by \3).